chainer/functions/softplus.py

```diff
@@ -0,0 +1,77 @@
+import numpy
+
+from chainer import cuda
+from chainer import function
+from chainer.utils import type_check
+
+
+class Softplus(function.Function):
+
+    """Softplus function."""
+
+    def __init__(self, beta=1.0):
+        self.beta = numpy.float32(beta)
+        self.beta_inv = numpy.float32(1.0 / beta)
+
+    def check_type_forward(self, in_types):
+        type_check.expect(in_types.size() == 1)
+        x_type, = in_types
+
+        type_check.expect(
+            x_type.dtype == numpy.float32,
+        )
+
+    def check_type_backward(self, in_types, out_types):
+        type_check.expect(
+            in_types.size() == 1,
+            out_types.size() == 1,
+        )
+        x_type, = in_types
+        g_type, = out_types
+
+        type_check.expect(
+            g_type.dtype == numpy.float32,
+            x_type.shape == g_type.shape,
+        )
+
+    def forward_cpu(self, inputs):
+        x, = inputs
+        # y = log(1 + exp(beta * x)) / beta
```

Comment:
I did not know softplus function whose beta is other than 1. I would like to know the reference of this formula.